mlir/lib/Conversion/FuncToEmitC/FuncToEmitC.cpp

```diff
@@ -38,7 +38,7 @@ class CallOpConversion final : public OpConversionPattern<func::CallOp> {
 
     rewriter.replaceOpWithNewOp<emitc::CallOp>(
         callOp,
-        callOp.getNumResults() ? callOp.getResult(0).getType() : nullptr,
```

Comment:
From reading the code, this seems to only be reached when numResults <= 1. Wouldn't `callOp.getResultTypes()` work here?

Reply:
Good idea